.github/workflows/update-manpage.yml

```diff
@@ -0,0 +1,65 @@
+name: Update manpage and completions
+
+on:
```

Comment:
Shouldn't this workflow run on schedule? Or at least limit it to the master branch.

Reply:
It think it's good to run on a schedule but it'd be good to have it run on PRs that change these files too (and the workflow file) so that they can be tested. 

Thoughts on adding these in the existing scheduled job?

Reply:
Maybe I'm misunderstanding, but what is the benefit of running on a schedule over on a push? It makes sense to me that we would want to run immediately after a push to create a new PR as soon as possible. If we're doing that consistently, I don't see a need for a scheduled job?

Reply:
> Maybe I'm misunderstanding, but what is the benefit of running on a schedule over on a push?

The manpage also relies on content from other repositories that you cannot rely on a push here triggering. You could use a push even for the other repositories perhaps though.

Reply:
> Thoughts on adding these in the existing scheduled job?

Am I right in assuming we'd want to have just a single `scheduled.yml`, for both the Sorbet and manpage updates?

> It think it's good to run on a schedule but it'd be good to have it run on PRs that change these files too (and the workflow file) so that they can be tested.

Currently the Sorbet update workflow runs on pushes that modify the workflow file, except pushes to `master` (apart from the scheduled runs). I'm probably missing something, but wouldn't we want this workflow to run on pushes to `master` that modify files in `cmd/` or `dev-cmd/` that would affect the manpages?

Reply:
> Am I right in assuming we'd want to have just a single `scheduled.yml`, for both the Sorbet and manpage updates?

Ideally, yes 👍🏻 



> I'm probably missing something, but wouldn't we want this workflow to run on pushes to `master` that modify files in `cmd/` or `dev-cmd/` that would affect the manpages?

I guess it depends if we want it committing directly to `master` or committing to the PR. I think it'll probably need to be the latter.